wiki/Featured_Artists/en.md

```diff
@@ -0,0 +1,13 @@
+# Featured Artists
+
+<!-- TODO: include some history about featured artists / mappers' guild, maybe in /History_of_osu! too -->
+
+**Featured artists** are music artists that have worked with osu! to make a selection of their tracks available for unrestricted use in the community. Some of the featured artists have also composed tracks specifically for osu! and released them as osu! exclusives.
```

Comment:
```suggestion
**Featured artists** are music artists that have made an agreement with osu! to make a selection of their tracks available for unrestricted use in the community. Some of the featured artists have also composed tracks specifically for osu! and released them as osu! exclusives.
```

Reply:
is the difference important? i think "worked with" sounds more natural